src/utils/content_protection_utils/inc/content_protection_utils.h

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file content_protection_utils.h
+ * @brief Provides an implementation for interacting with the content protection section of a deployment's json
+ *
+ * @copyright Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+#include <aduc/c_utils.h>
+#include <aduc/decryption_alg_types.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#ifndef CONTENT_PROTECTION_UTILS_H
+#    define CONTENT_PROTECTION_UTILS_H
+
+EXTERN_C_BEGIN
+
+bool ContentProtectionUtils_DecryptDek(const STRING_HANDLE dek, const STRING_HANDLE dekCryptAlg);
+
+bool ContentProtectionUtils_GetDecryptAlgFromDecryptionInfo(const JSON_Object* decryptInfo, DecryptionAlg* alg);
```

Comment:
why bool? returning "enum DecryptionAlg" seems more logical to me, and have DecryptionAlg have a INVALID field that indicates error?